float32[100] classes
uint16 num_tracked
Box[100] boxes
string[100] track_ids
